modules/ecs-service/README.md

```diff
@@ -7,21 +7,144 @@ The following requirements are needed by this module:
-- [aws_ecs_task_definition.task](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition) (resource)
-- [aws_service_discovery_service.registry](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/service_discovery_service) (resource)
+	 # --------------------------------------------
+	 # Required variables
+	 # --------------------------------------------
 
+
+	 # ARN of an ECS cluster.
+	 cluster_arn  = string
+
+
+	 # The ID of the namespace to use for DNS configuration.
+	 dns_namespace_id  = string
+
+
+	 # The image used to start a container. This string is passed directly to the Docker daemon. Images in the Docker Hub registry are available by default. You can also specify other repositories with either `repository-url/image:tag` or `repository-url/image@digest`. Up to 255 letters (uppercase and lowercase), numbers, hyphens, underscores, colons, periods, forward slashes, and number signs are allowed.
+	 image  = string
+
+
+	 # Name that will associate all resources.
+	 service_name  = string
+
+
+	 # --------------------------------------------
+	 # Optional variables
+	 # --------------------------------------------
+
+
+	 # The hard limit of CPU units to present for the task. For tasks that use the Fargate launch type (both Linux and Windows containers), this field is required. 
+	 container_cpu  = number
+
+
+	 # The amount (in MiB) of memory to present to the container. [container_definition_memory](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_memory)
+	 container_memory  = number
+
+
+	 # The port number on the container that's bound to the user-specified or automatically assigned host port.
+	 container_port  = number
+
+
+	 # Must be set to either `X86_64` or `ARM64`; see cpu architecture.
+	 cpu_architecture  = string
+
+
+	 # Number of instances of the task definition to place and keep running. Defaults to 1. Do not specify if using the `DAEMON` scheduling strategy.
+	 desired_count  = number
+
+
+	 # Set to true if service discovery should not be configured. Otherwise either set the service_registries block or provide values for service_name, and container_port.
+	 disable_service_discovery  = bool
+
+
+	 # The environment variables to pass to a container. This parameter maps to the --env option to docker run. Consists (name, value)
+	 environment  = list(any)
+
+
+	 # A list of files containing the environment variables to pass to a container. This parameter maps to the `--env-file` option to `docker run`. Consists (value, type = "s3")
+	 environment_files  = list(any)
+
+
+	 # Ephemeral storage block, consists (size_in_gib): The minimum supported value is `21` GiB and the maximum supported value is `200` GiB. This parameter is used to expand the total amount of ephemeral storage available, beyond the default amount, for tasks hosted on AWS Fargate. See main.tf
```

Comment:
![image](https://github.com/Cyber4All/terraform-cyber4all-catalog/assets/55514493/dcad01b0-edae-48ab-9aa4-1c7e95c79881)
Good times in bundling

Reply:
```suggestion
	 # Ephemeral storage block, consists (sizeInGiB): The minimum supported value is `21` GiB and the maximum supported value is `200` GiB. This parameter is used to expand the total amount of ephemeral storage available, beyond the default amount, for tasks hosted on AWS Fargate. See main.tf
```
Just wanted to point this var out and check if the reference was correct. Will all vars in terraform have the underscore? Or does it just depend?

Reply:
All will have the underscores.